lib/puppet/rest/routes.rb

```diff
@@ -22,5 +22,15 @@ def self.get_certificate(client, name)
       end
       body
     end
+
+    def self.get_crls(client, name)
+      header = { 'Accept' => 'text/plain', 'accept-encoding' => ACCEPT_ENCODING }
```

Comment:
Also, I missed this in another review, but headers are conventionally title cased, but also case insensitive. So I would prefer to see 'Accept' and 'Accept-Encoding' as keys.